project/Settings.scala

```diff
@@ -144,8 +143,7 @@ object Settings {
   //thus commonSettings needs to be added first.
   val rootSettings = commonSettings ++ List(
     name := "rawls",
-    version := "0.1"
-  ) ++ rawlsAssemblySettings ++ noPublishSettings ++ rawlsCompileSettings ++ java17BuildSettings
+  ) ++ versionSettings ++ rawlsAssemblySettings ++ noPublishSettings ++ rawlsCompileSettings ++ java17BuildSettings
```

Comment:
not sure why `rawlsCoreSettings` and `rootSettings` overrode the "version" key to be 0.1, instead of reading the calculated version from `Version.scala`. Updated to read the calculated version.